src/tree/index.js

```diff
@@ -134,7 +135,7 @@ class Tree extends Component {
       /* https://www.w3.org/TR/wai-aria-1.1/#select
        * https://www.w3.org/TR/wai-aria-1.1/#tree */
       role: mode === 'simpleSelect' ? 'listbox' : 'tree',
-      'aria-multiselectable': /multiSelect|hierarchical/.test(mode),
+      'aria-multiselectable': /multiSelect|hierarchical/.test(mode).toString(),
```

Comment:
Do we need the `toString()` or is it from before our convo on the other PR?

Reply:
Yes. It will only be appended on true otherwise.

Reply:
Right, but omitting that is ok. See https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/listbox_role